modules/bmo-getting-the-baremetalhost-resource.adoc

```diff
@@ -115,7 +114,6 @@ status:
       manufacturer: HPE
       productName: ProLiant DL380 Gen10 (868703-B21)
       serialNumber: CZ200606M3
-  hardwareProfile: unknown
```

Comment:
The profile is still present in the status (since we haven't fully removed it). Not sure if you want to keep it for completeness.

Reply:
Updated As Suggested.